src/Arcus.Scripting.ApiManagement/Scripts/Remove-AzApiManagementDefaults.ps1

```diff
@@ -5,32 +5,71 @@ param(
 
 Write-Host "Start removing Azure API Management defaults..."
 $apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroupName -ServiceName $ServiceName 
+$exceptionOccurredOnDelete = $false
 
-Write-Host "Removing Echo Api..."
-$apiResult = Remove-AzApiManagementApi -Context $apimContext -ApiId 'echo-api'
-
-Write-Host "Removing Starter product..."
-$starterResult = Remove-AzApiManagementProduct -Context $apimContext -ProductId 'starter' -DeleteSubscriptions
-
-Write-Host "Removing Unlimited product..."
-$unlimitedResult = Remove-AzApiManagementProduct -Context $apimContext -ProductId 'unlimited' -DeleteSubscriptions
+Write-Host "Checking if 'echo' API exists..."
+$echoExists = $true
+try {
+    $apiGetResult = Get-AzApiManagementApi -Context $apimContext -ApiId 'echo-api' -ErrorAction Stop
```

Comment:
We don't do anything with these results, right?

Reply:
Correct, if I remove this and the GETS return a response we get all the info in the logging, for example:
![image](https://user-images.githubusercontent.com/32359437/147456234-e7615301-76e0-45c4-be91-819c3be80a43.png)

To me this seemed a bit overkill to write to the log as it gets a bit convoluted, I would rather have something like this instead of including to information from the GET:
```
Checking if 'starter' product exists...
Removing 'starter' product...
Checking if 'unlimited' product exists...
```

And by including the `$apiGetResult = ` I don't get the output from the GET action. 
I switched it to `Out-Null` (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/out-null?view=powershell-7.2), that's probably a better implementation to prevent all the logging info. See [`f7632b2` (#254)](https://github.com/arcus-azure/arcus.scripting/pull/254/commits/f7632b239c90bfa4bbc3e1a78787f1c5dbac448f)